configure

```diff
@@ -66,7 +66,7 @@ while true; do
       ;;
 
     --readline=*)
-      FLAG_readline=$(expr "$1" : '--readline=\(.*\)')
+      FLAG_readline=$(expr \( "$1" : '--readline=\(.*\)' \) )
```

Comment:
Please add a comment saying that `\(` is necessary for FreeBSD (which is honestly a great example of sucky language design in shell tools).

And link to the manual  too:

https://www.freebsd.org/cgi/man.cgi?query=expr&sektion=&n=1